packages/frontend/src/pages/Login.tsx

```diff
@@ -130,20 +130,19 @@ const Login: FC = () => {
                             style={{
                                 marginTop: 20,
                                 display: 'flex',
-                                justifyContent: 'flex-end',
+                                justifyContent: 'space-between',
+                                alignItems: 'center',
                             }}
                         >
                             {health.data?.hasEmailClient && (
-                                <LinkButton
+                                <AnchorLink
                                     href="/recover-password"
-                                    minimal
                                     style={{
-                                        marginRight: 10,
-                                        color: Colors.GRAY1,
+                                        color: Colors.BLUE4,
                                     }}
```

Comment:
Do we need to set the blue colour ? Should we set this in the AnchorLink component so they all look the same ?